lib/rubocop/cop/rspec/no_context.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Don't use context blocks.
+      #
+      # @example
+      #   # bad
+      #   context 'display name presence' do
+      #     # ...
+      #   end
+      #
+      #   # good
+      #   describe 'display name presence' do
+      #     # ...
+      #   end
+      class NoContext < Cop
+        MSG = "Don't use context blocks.".freeze
+
+        def_node_matcher :context_block, <<-PATTERN
+          (block (send _ { :context } _) ...)
```

Comment:
you don't need the curly braces around the method name, as they act as an OR but you have only one option

Reply:
It's guaranteed to produce false positives for methods that are named context and accept a block. That's why we check for either `nil?` or `(const nil? RSpec)` receiver